openedx/core/djangoapps/credit/models.py

```diff
@@ -134,6 +134,24 @@ def disable_credit_requirements(cls, requirement_ids):
         cls.objects.filter(id__in=requirement_ids).update(active=False)
 
 
+    @classmethod
+    def get_course_requirement(cls, course_key, namespace, name): #TODO temporary methods
```

Comment:
why `#TODO temporary methods`? Please add some minimal description so that it become something understandable.